google-cloud-logging/src/test/java/com/google/cloud/logging/LoggingImplTest.java

```diff
@@ -113,6 +117,12 @@
   private static final String LOG_NAMES_CURSOR = "cursor";
   private static final String LOG_NAME = "log";
   private static final String LOG_NAME_PB = "projects/" + PROJECT + "/logs/" + LOG_NAME;
+  private static final String LOG_NAME_OVERRIDE_PB =
```

Comment:
I am not sure whether you followed existing pattern or not but `_PB` suffix is not informative IMO. The constant value is a fully qualified name but it is not related to protobuf (assuming `_PB` suffix references it).
I recommend to review all constants with this suffix and make their names to be more informative. In this particular case `LOG_NAME` can be renamed to `LOG_ID` and `LOG_NAME_PB` to become `LOG_NAME`.

Reply:
As for LOG_NAME - it is used in calls to setLogName(), so passing LOG_ID might be a bit misleading. I changed some names, please let me know if that OK